pkg/kubectl/cmd/util/helpers.go

```diff
@@ -46,6 +46,7 @@ import (
 
 const (
 	ApplyAnnotationsFlag = "save-config"
+	AddOnLabel           = "kubernetes.io/cluster-service"
```

Comment:
This PR only plans to check if a resource is a cluster service, and then set "write-protect" flag accordingly. "kubernetes.io/cluster-service" is the existing label for identifying cluster services. Currently we don't have such a label for write-protected resources.